ark/utils/data_utils.py

```diff
@@ -544,3 +546,29 @@ def split_img_stack(stack_dir, output_dir, stack_list, indices, names, channels_
 
             save_path = os.path.join(img_dir, names[i])
             io.imsave(save_path, channel, plugin='tifffile', check_contrast=False)
+
+
+def download_example_data(save_dir: Union[str, pathlib.Path],
+                          get_data_path: bool = True) -> Optional[pathlib.Path]:
+    """Downloads the example dataset from Hugging Face Hub.
+    The following is a link to the dataset used:
+    https://huggingface.co/datasets/angelolab/ark_example
+
+    The dataset will be saved in `save_dir/example_dataset/raw`.
+
+    Args:
+        save_dir (Union[str, pathlib.Path]): The directory to save the example in
+        get_data_path (bool): Returns the path of the dataset directory in order to load the
+            example data. Defaults to True.
+
+    Returns:
+        Optional[pathlib.Path]: Returns the path where the dataset is saved.
+    """
+    cache_dir = pathlib.Path(save_dir) / "example_dataset" / "raw"
```

Comment:
Any reason to hardcode this? Why not have it passed as an argument?

Reply:
No reason to hardcode it, I'll add them as arguments.